Sources/VLCEmptyLibraryView.m

```diff
@@ -21,6 +22,13 @@ - (void)awakeFromNib
     _emptyLibraryLabel.text = NSLocalizedString(@"EMPTY_LIBRARY", nil);
     _emptyLibraryLongDescriptionLabel.text = NSLocalizedString(@"EMPTY_LIBRARY_LONG", nil);
     [_learnMoreButton setTitle:NSLocalizedString(@"BUTTON_LEARN_MORE", nil) forState:UIControlStateNormal];
+    
+    NSNotificationCenter *notificationCenter = [NSNotificationCenter defaultCenter];
+    [notificationCenter addObserver:self
```

Comment:
I think we don't need to cache the default center here since it's only used once!
So I would suggest to directly use `[[NSNotificationCenter defaultCenter] addObserver:...]`